.gitignore

```diff
@@ -72,3 +72,5 @@ libmdbx/build/*
 tests/testdata/*
 
 go.work
+
+/goerli
```

Comment:
Likely it's because you did --datadir=goerli
Storing data inside source directory is bad practice, do something like --datadir=/home/alex/data/goerli

Reply:
Should I remove this line from .gitignore then?